packages/select/src/styles/vaadin-select-overlay-base-styles.js

```diff
@@ -25,21 +25,26 @@ export const selectOverlayStyles = css`
 
   :host([theme~='align-start']) [part='overlay'] {
     text-align: start;
+    --vaadin-item-text-align: start;
```

Comment:
I think we don't need `text-align` anymore with this change as it does not take any effect on items, so maybe it can be removed in favor of `--vaadin-item-text-align`?

Also, custom CSS property can be set on the `:host([theme~='align-start'])` without `[part='overlay']`.